README.md

```diff
@@ -1,6 +1,7 @@
 # NTP Mock Server
 
-[![Build Status](https://travis-ci.org/dmcnulla/ntp.svg?branch=master)](https://travis-ci.org/dmcnulla/ntp)
+[![Build Status](https://travis-ci.org/dmcnulla/ntp.svg?branch=coverage)](https://travis-ci.org/dmcnulla/ntp)
```

Comment:
yes, I see, but if you merges coverage into the master branch, the changes will belong to coverage anyway.